iam-login-service/src/main/java/it/infn/mw/iam/api/scim/updater/factory/DefaultAccountUpdaterFactory.java

```diff
@@ -208,6 +208,10 @@ private void prepareReplacers(List<AccountUpdater> updaters, ScimUser user, IamA
     addUpdater(updaters, Objects::nonNull, user::getPassword, replace::password);
     addUpdater(updaters, Objects::nonNull, user::getActive, replace::active);
 
+    if (user.hasServiceAccountStatus()) {
+      addUpdater(updaters, Objects::nonNull, user::getServiceAccount, replace::serviceAccount); 
```

Comment:
here it becomes `user.getIndigoUser()::getServiceAccount`

Reply:
Done